apps/charge-accounts-service/src/operators/operators.service.ts

```diff
@@ -160,25 +176,30 @@ export class OperatorsService {
     if (!operatorWalletCreationResult) {
       throw new HttpException('Failed to create operator wallet', HttpStatus.INTERNAL_SERVER_ERROR)
     }
+    return operatorWalletCreationResult
   }
 
-  private constructUserProjectResponse (user: any, projectObject: any, publicKey: string, secretKey: string, sponsorId: string) {
-    // Constructs the response object from the created entities
+  private constructUserProjectResponse (user: any, project: any, wallet: any, secretKey?: string) {
+    // Constructs the response object from the entities
```

Comment:
Create interface also for this function please

Reply:
@NikolayRn updated